lib/utils/__tests__/isStandardSyntaxUrl.test.js

```diff
@@ -107,16 +107,16 @@ describe("isStandardSyntaxUrl", () => {
     expect(isStandardSyntaxUrl("some/@url")).toBeTruthy();
   });
   it("url with less interpolation without quotes", () => {
-    expect(isStandardSyntaxUrl("@{less-variable}")).toBeTruthy();
+    expect(isStandardSyntaxUrl("@{less-variable}")).toBeFalsy();
   });
   it("url with less interpolation at the start without quotes", () => {
-    expect(isStandardSyntaxUrl("@{less-variable}/path")).toBeTruthy();
+    expect(isStandardSyntaxUrl("@{less-variable}/path")).toBeFalsy();
   });
   it("url with less interpolation in the middle without quotes", () => {
-    expect(isStandardSyntaxUrl("some/@{less-variable}/path")).toBeTruthy();
+    expect(isStandardSyntaxUrl("some/@{less-variable}/path")).toBeFalsy();
   });
   it("url with less interpolation at the end without quotes", () => {
-    expect(isStandardSyntaxUrl("some/@{less-variable}")).toBeTruthy();
+    expect(isStandardSyntaxUrl("some/@{less-variable}")).toBeFalsy();
```

Comment:
Looks like a regression

Reply:
Actually, it's a bug fix. Interpolation makes URL non-standard. There are tests for Sass interpolation and they all negative.

Reply:
:+1: